php_phongo.c

```diff
@@ -3195,6 +3199,33 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_row(2, "libmongoc compression", "disabled");
 #endif
 
+#ifdef MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION
+#ifdef HAVE_SYSTEM_LIBMONGOCRYPT
+	php_info_print_table_row(2, "libmongocrypt headers version", MONGOCRYPT_VERSION);
+	php_info_print_table_row(2, "libmongocrypt library version", mongocrypt_version());
+#else
+	php_info_print_table_row(2, "libmongocrypt bundled version", MONGOCRYPT_VERSION);
+#endif
+
+#ifdef MONGOCRYPT_ENABLE_CRYPTO
+	php_info_print_table_row(2, "libmongocrypt crypto", "enabled");
+
+#if defined(MONGOC_ENABLE_CRYPTO_OPENSSL)
+	php_info_print_table_row(2, "libmongocrypt crypto library", "OpenSSL");
+#elif defined(MONGOC_ENABLE_CRYPTO_COMMON_CRYPTO)
+	php_info_print_table_row(2, "libmongocrypt crypto library", "Common Crypto");
+#elif defined(MONGOC_ENABLE_CRYPTO_CNG)
+	php_info_print_table_row(2, "libmongocrypt crypto library", "CNG");
+#else
+	php_info_print_table_row(2, "libmongocrypt crypto library", "unknown");
+#endif
+#else
+	php_info_print_table_row(2, "libmongocrypt crypto", "disabled");
+#endif
+#else
```

Comment:
I'd suggest adding `/* MONGOCRYPT_ENABLE_CRYPTO */` here, as we have above for libmongoc's crypto. Helps readability for long `#if` chains.

Reply:
Added a comment, especially since this is not the `#else` for `MONGOCRYPT_ENABLE_CRYPTO` but rather `MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION`. It beautifully proves your point.